### DISCUSSION TITLE
Table 12, Panel 1

### DISCUSSION THREAD

**Discussion by githubyongchen** [cssi-pi-community/2022-meeting#35] [Panel 1 roundtable (Monday morning)]
We believe the work-from-home policy can be attractive for women and minority candidates; however some universities ban this policy. Would NSF be able to do something to make work policy more flexible/friendly for minority population?

**Comment by klehnert55** [comment]
We need better statistics / metrics to track the status of diversity in CI as compared to other domain and program

**Comment by klehnert55** [comment]
Retention is a problem. Minority students want to get MS and leave for industry.
